src/main/java/mate/academy/hibernate/relations/model/Movie.java

```diff
@@ -1,11 +1,23 @@
 package mate.academy.hibernate.relations.model;
 
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.OneToMany;
+import jakarta.persistence.Table;
 import java.util.ArrayList;
 import java.util.List;
 
+@Entity
+@Table(name = "movies")
 public class Movie implements Cloneable {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
     private String title;
+    @OneToMany(fetch = FetchType.LAZY)
```

Comment:
I think initializing Fetch.Type.Lazy is redundant

Reply:
it is redundant for `*-to-many` relations, but it can also stay to be that explicit, up to you

Reply:
But the main thing here - is it a proper type of relationship?
How many actors can play in a single movie?
How many movies can 1 actor act in?
What would be a relation between them, judjing by the answers to those 2 last questions? always ask them yourself if you are not sure.

We also need to specify the `JoinColumn`/`JoinTable` depending on what relation you have, that goes for other entities as well